terraform/registry/README.md

```diff
@@ -0,0 +1,78 @@
+# sunodo s3 deployment
```

Comment:
Use `markdownlint` here, and fix the problems.

I'll do another review when the linting is done.

Reply:
Fixed !